src/libLookup/Lookup.cpp

```diff
@@ -1389,12 +1410,13 @@ bool Lookup::ProcessSetDSInfoFromSeed(const vector<unsigned char>& message,
 
   LOG_MARKER();
 
+  bool initialDS;
```

Comment:
Please initilize the variable when define

Reply:
ok